istio-proxy/fetch.sh

```diff
@@ -265,13 +277,13 @@ function patch_class_memaccess() {
 function replace_ssl() {
   if [ "$REPLACE_SSL" = "true" ]; then
     pushd ${PROXY_FETCH_DIR}/proxy
-      git clone http://github.com/maistra/istio-proxy-openssl -b ${OPENSSL_GIT_BRANCH}
+      git clone http://github.com/bdecoste/istio-proxy-openssl -b ${OPENSSL_GIT_BRANCH}
```

Comment:
hmm. I think this should be pointing to https://github.com/Maistra/istio-proxy-openssl

Reply:
ugh, Bill's 1.1 branch has 1.3-related commits, which will need to be moved to Maistra/istio-proxy-openssl